Cooper/DAO/Mapping/EntityMapping.cs

```diff
@@ -212,7 +286,192 @@ public static EntityORM Map(UserDb user, HashSet<string> attributes)
+        ///</summary>
+        public static void Map(EntityORM entity, out ChatDb chat)
+        {
+            chat = new ChatDb();
+
+            foreach (KeyValuePair<string, object> aV in entity.attributeValue)
+            {
+                switch (aV.Key)  // entity attribute
+                {
+                    case "ID":
+                        chat.Id = Convert.ToInt64(aV.Value);
+                        break;
+                    case "CHATNAME":
+                        chat.ChatName = aV.Value.ToString();
+                        break;
+                    default:
+                        break;
+                }
+            }
+        }
+
+        ///<summary>
+        ///Maps properties from ChatDb object to EntityORM object
+        ///</summary>
+        public static EntityORM Map(ChatDb chat, HashSet<string> attributes)
+        {
+            EntityORM entity = new EntityORM();
+
+            foreach (string attribute in attributes)
+            {
+                object value = null;        // attribute value
+
+                switch (attribute)
+                {
+                    case "CHATNAME":
+                        value = $"\'{chat.ChatName}\'";
+                        break;
+                    default:
+                        break;
+                }
+
+                entity.attributeValue.Add(attribute, value);
+            }
+
+            return entity;
+        }
+        #endregion
+        
+
+
+        #region Helping methods
+
+        private static string ToOracleDateFormat(DateTime dateTime)
+        {
+            string day = dateTime.Day.ToString();
+            string month = dateTime.Month.ToString();
+            string year = dateTime.Year.ToString();
+
+            switch (month)
+            {
+                case "1":
+                    month = "JAN";
+                    break;
+                case "2":
+                    month = "FEB";
+                    break;
+                case "3":
+                    month = "MAR";
+                    break;
+                case "4":
+                    month = "APR";
+                    break;
+                case "5":
+                    month = "MAN";
+                    break;
+                case "6":
+                    month = "JUN";
+                    break;
+                case "7":
+                    month = "JUL";
+                    break;
+                case "8":
+                    month = "AUG";
+                    break;
+                case "9":
+                    month = "SEP";
+                    break;
+                case "10":
+                    month = "OCT";
+                    break;
+                case "11":
+                    month = "NOV";
+                    break;
+                case "12":
+                    month = "DEC";
+                    break;
+            }
+
+            string oracleDateFormat = $"{day}-{month}-{year}";
```

Comment:
dt.ToString("dd-MMM-yyyy")

Reply:
Removed corresponding method for converting date